Main.java

```diff
@@ -9,7 +10,30 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+  // create a Scanner for user input
+  Scanner input = new Scanner(System.in);
+    
+   
+    //initialize a number that needs to be added to another number
+
+        // declare a variable for chromebook chargers
+  double chromebookCharger = (34.99*3);
+
+  // declare a variable for motherboards
+  double motherboards = (127.5*2);
+
+//declare a variable for mouses
+  double mouse = (18*5);
```

Comment:
The 3, 2, and 5 are all numbers that should be input by the user. This should not be hard coded in!